### PR TITLE
feat(dev): control halyard configuration when building images.

### PR DESCRIPTION
**Pull request by ewiseblatt**
@jtk54

### CODE REVIEW

File: dev/halyard_install_component.sh [hunk 1]
```diff
@@ -72,11 +88,15 @@ function process_args() {
             shift
             ;;
 
-        --halyard_repository)
-            HALYARD_INSTALL_PARAMS="$HALYARD_INSTALL_PARAMS --halyard-repository $1"
+        --spinnaker_registry)
```
Comment (jtk54) [inline dev/halyard_install_component.sh]: Only two of the new flags added to the `--help` seem to be implemented.

Comment (jtk54) [reply dev/halyard_install_component.sh]: Or do these get delegated to the python script?

Comment (ewiseblatt) [reply dev/halyard_install_component.sh]: They are passed to the install script at line 119

Comment (jtk54) [reply dev/halyard_install_component.sh]: Got it.

**Pull request merged**